server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -642,7 +643,7 @@ private void getMonitorLock() throws KeeperException, InterruptedException {
       UtilWaitThread.sleep(getContext().getConfiguration().getTimeInMillis(Property.MONITOR_LOCK_CHECK_INTERVAL));
     }
 
-    log.info("Got Monitor lock.");
+    log.info("Acquired Monitor Lock " + monitorLock.getLockPath());
```

Comment:
@lstav check out https://www.slf4j.org/faq.html#logging_performance.  It's better because the string concatenation doesn't happen if the logger is not going to output.